src/main/python/api.py

```diff
@@ -0,0 +1,44 @@
+from flask import Flask, jsonify, request
+import argparse
```

Comment:
For style, standard library imports should go before third party library imports. See https://www.python.org/dev/peps/pep-0008/#imports.

So we should have:

```python
import argparse

from flask import Flask, jsonify, request
...
```